### PR TITLE
Added notes on how to find waiter info

### PR DESCRIPTION
**Pull request by SamRemis**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/guide_waiters.rst [hunk 1]
```diff
@@ -19,7 +19,12 @@ Waiters in the |sdk-php| Version 3
 Waiters help make it easier to work with *eventually consistent* systems by
 providing an abstracted way to wait until a resource enters into a particular
 state by polling the resource. You can find a list of the waiters supported by
-a client by viewing the API documentation of a service client.
+a client by viewing the API documentation for a single version of a service client.
+To navigate there, go to the client's page in our documentation and click on the
```
Comment (howardlopez) [inline doc_source/guide_waiters.rst:23]: Should this specify "API documentation" to distinguish from the dev guide documentation? Also, should "API documentation" itself be a link to the API docs homepage? https://docs.aws.amazon.com/aws-sdk-php/v3/api/index.html

File: doc_source/guide_waiters.rst [hunk 2]
```diff
@@ -19,7 +19,12 @@ Waiters in the |sdk-php| Version 3
 Waiters help make it easier to work with *eventually consistent* systems by
 providing an abstracted way to wait until a resource enters into a particular
 state by polling the resource. You can find a list of the waiters supported by
-a client by viewing the API documentation of a service client.
+a client by viewing the API documentation for a single version of a service client.
+To navigate there, go to the client's page in our documentation and click on the
+hyperlink that is in the date of the version and scroll down to the 'Waiters' section.
```
Comment (howardlopez) [inline doc_source/guide_waiters.rst:24]: "click on the hyperlink that is in the date of the version" may be a bit confusing. Perhaps something like "navigate to a specific API version (represented by a date)"?

File: doc_source/guide_waiters.rst (hunk 1 above)
Comment (SamRemis) [reply doc_source/guide_waiters.rst]: I updated the language to more clearly reflect which docs i was referring to and added the link.  Thank you

File: doc_source/guide_waiters.rst (hunk 2 above)
Comment (SamRemis) [reply doc_source/guide_waiters.rst]: Updated the language to "To navigate there, go to the client's
page in the API documentation and navigate to the specific version number
(represented by a date) and scroll down to the 'Waiters' section."

File: doc_source/guide_waiters.rst [hunk 3]
```diff
@@ -19,7 +19,11 @@ Waiters in the |sdk-php| Version 3
 Waiters help make it easier to work with *eventually consistent* systems by
 providing an abstracted way to wait until a resource enters into a particular
 state by polling the resource. You can find a list of the waiters supported by
-a client by viewing the API documentation of a service client.
+a client by viewing the `API documentation <https://docs.aws.amazon.com/aws-sdk-php/v3/api/index.html>`_
+for a single version of a service client.To navigate there, go to the client's
```
Comment (howardlopez) [inline doc_source/guide_waiters.rst:23]: Looks like there should be a space between the period and `To`.

**Pull request merged**